docs/data/joy/customization/theme-tokens/theme-tokens.md

```diff
@@ -0,0 +1,351 @@
+# Theme tokens
+
+<p class="description">Learn how to customize the theme tokens.</p>
+
+<!-- This page should answer: -->
+<!-- What's the structure -->
+<!-- How to override, and how to augment the type -->
+<!-- (optional) live playground -->
+
+## Override default tokens
+
+To customize the default theme tokens, you have to use `extendTheme` API to create a theme and then pass it to `CssVarsProvider`. The specified tokens will be deeply merged with the default values. The value can also be a CSS variable that reference to another one:
```

Comment:
```suggestion
To customize the default theme tokens, you have to use `extendTheme` API to create a theme and then pass it to `CssVarsProvider`.
The specified tokens will be deeply merged with the default values.
```

Let's explain one thing at a time. We can add a second demo for this or expand on it later.